models/staging/src_fivetran_platform.yml

```
@@ -200,5 +200,29 @@ sources:
             description: Phone number associated with user.
           - name: verified 
             description: Boolean that indicates whether the user has verified their email address in the account creation process.
+          - name: _fivetran_synced
+            description: Timestamp of when the record was last synced.
+
+      - name: transformation_runs
+        identifier: "{{ var('fivetran_log_transformation_runs_identifier', 'transformation_runs') }}"
```

Looks like we are using the old name here. We need to update this to platform.
```suggestion
        identifier: "{{ var('fivetran_platform_transformation_runs_identifier', 'transformation_runs') }}"
```

---

We reference the platform name in the platform named version [here](https://github.com/fivetran/dbt_fivetran_log/blob/9f095e8939740dfe8cab0e6084aef5cc1736cbbb/integration_tests/dbt_project.yml#L23) as well, so this will be consistent